src/Functions/FunctionsBitmap.h

```diff
@@ -1176,10 +1176,14 @@ class FunctionBitmap : public IFunction
             const AggregateDataPtr data_ptr_0 = is_column_const[0] ? container0[0] : container0[i];
             const AggregateDataPtr data_ptr_1 = is_column_const[1] ? container1[0] : container1[i];
 
-            col_to->insertFrom(data_ptr_0);
+            auto && bm_1 = reinterpret_cast<AggregateFunctionGroupBitmapData<T> *>(data_ptr_0);
+            auto && bm_2 = reinterpret_cast<AggregateFunctionGroupBitmapData<T> *>(data_ptr_1);
+
+            auto need_exchange = (name == NameBitmapAnd::name) && bm_1->rbs.isLarge() && bm_2->rbs.isSmall();
```

Comment:
Worth adding a comment here : why we exchange the arguments.

Reply:
ok, no problem